app/controllers/articles_controller.rb

```diff
@@ -16,6 +16,10 @@ def create
     end
   end
 
+  def index
+    @articles = Article.order('created_at DESC')
```

Comment:
**Not a blocker:** Although this syntax is correct, and it's even part of the [order method documentation](https://apidock.com/rails/ActiveRecord/QueryMethods/order), you should be aware that it's considered a best practice to use `:symbols` when possible.